source/cookies.html.erb

```diff
@@ -0,0 +1,96 @@
+---
+title: Cookies - GOV.UK Pay
+---
+
+<nav class="breadcrumbs" aria-label="Breadcrumbs">
+  <ol itemscope itemtype="http://schema.org/BreadcrumbList">
+    <li class="breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+      <a href="https://www.gov.uk/service-toolkit#components" itemprop="item"><span itemprop="name">Components</span></a>
+    </li>
+    <li class="breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+      <a href="/" itemprop="item"><span itemprop="name">GOV.UK Pay</span></a>
+    </li>
+    <li class="breadcrumbs__item breadcrumbs__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+      <a href="#main" itemprop="item"><span itemprop="name">Cookies</span></a>
+    </li>
+  </ol>
+</nav>
+
+<div class="container">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 id="heading-1">Cookies</h1>
+      <p class="summary">
+        GOV.UK Pay puts small files (known as ‘cookies’)
+        on to your computer.
+      </p>
+      <p>Cookies are used to:</p>
+      <ul class="list list-bullet">
+        <li>remember you once you've signed in</li>
+        <li>remember the notifications you've seen so we don't show them again</li>
+        <li>measure how you use the website so it can be updated and improved based on your needs</li>
+      </ul>
+
+      <p>You’ll normally see a message on the site before we store a cookie on your computer.</p>
+
+      <p>Find out <a href="http://www.aboutcookies.org/">how to manage cookies</a>.</p>
+
+      <h2 class="heading-medium">Session cookie</h2>
```

Comment:
I've updated the content on the cookie policy page to reflect the relevant cookies (payment pages do not need to be covered) in this ticket  PP-2434